src/main/java/frc/robot/pioneersLib/bumSwerve/Encoder/EncoderIO.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.pioneersLib.bumSwerve.Encoder;
+
+import org.littletonrobotics.junction.AutoLog;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public interface EncoderIO {
+
+    @AutoLog
+    public static class EncoderIOInputs {
+        public double absoluteEncoderOffset;
+        public boolean inverted;
+    }
+
+    public static class EncoderIOOutputs {
+        public double turnAbsolutePosition;
```

Comment:
this is an input I think??? idrk at this point. Shouldn't be any outputs for encoder